src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class CinemaHallRequestDto {
+    @NotNull
+    @Size(min = 20, max = 300)
```

Comment:
Can you use `Size` here?
![image](https://user-images.githubusercontent.com/32339836/236235585-27e808cf-d5bf-4a1a-98f0-ce18751824ac.png)